optuna/cli.py

```diff
@@ -223,12 +224,42 @@ def take_action(self, parsed_args):
         return 0
 
 
+class StorageUpgrade(BaseCommand):
+    def get_parser(self, prog_name):
+        # type: (str) -> ArgumentParser
+
+        parser = super(StorageUpgrade, self).get_parser(prog_name)
+        return parser
+
+    def take_action(self, parsed_args):
+        # type: (Namespace) -> None
+
+        config = optuna.config.load_optuna_config(self.app_args.config)
```

Comment:
As we discussed offline, please stop reading the optuna config to prevent unintended upgrade of databases.
And please add unit tests for this command.

Reply:
We discussed further offline and decided to allow config reading if `--config` option is explicitly specified.

Reply:
Fixed: https://github.com/pfnet/optuna/pull/336/commits/6e160eaf337c5565392050547da4d2be95e1b9a5